protocol/oracle-manager/contracts/interfaces/INodeModule.sol

```diff
@@ -16,6 +16,16 @@ interface INodeModule {
      */
     error InvalidNodeDefinition(NodeDefinition.Data nodeType);
 
+    /**
+     * @notice An array of revert reasons when an array of nodes is processed, but some of the nodes failed
+     */
+    error Errors(bytes[] revertReasons);
```

Comment:
```suggestion
    error OracleNodeErrors(bytes[] revertReasons);
```

Reply:
Its unknown for sure if the errors are going to be Oracle node related or some other error. so it makes more sense to just call them errors and let the client decide what type of errors they are and if there's any action that can or should be taken

Reply:
Oh, I see what you're saying with this suggestion. My concern is that we may find this feature to be useful and use it elsewhere. I think it would be good to treat this as something more standardized and know how to recursively parse groups of errors like this, not just for erc7412

Reply:
I can see that the same format used in other places. So maybe it is fine after all. Is it worth extracting as separate reusable interface maybe instead of duplicating?